content/trial_getting_started/introduction_to_blueprints/writing_blueprint.md

```diff
@@ -280,10 +265,7 @@ There are basically two mode in which this could be done:
           id: ex3-relationship-blueprint
 {{< /highlight >}}
 
-2. Upload and install a blueprint on demand
-
-If the blueprint wasn’t preloaded to the manager we can load it on demand.
-To do that we need to include _blueprint_archive_ which points to the package that contains this service and _main_file_name_ which points to the blueprint file in that package as described below:
+2. Uploading and installing a blueprint on demand. If the blueprint isn't already loaded in the manager, it can be uploaded on demand using the `blueprint_archive` and `main_file_name` parameter. These parameters point to the ZIP archive containing the blueprin and the main blueprint file name, respectively.
```

Comment:
```suggestion
2. Uploading and installing a blueprint on demand. If the blueprint isn't already loaded in the manager, it can be uploaded on demand using the `blueprint_archive` and `main_file_name` parameter. These parameters point to the ZIP archive containing the blueprint and the main blueprint file name, respectively.
```